Documentation/content/examples/index.md

```diff
@@ -63,32 +64,130 @@ This will allow you to see the some live code running in your browser. Just pick
+[![WindowedSincPolyDataFilter Example][WindowedSincPolyDataFilter]](./WindowedSincPolyDataFilter.html "WindowedSincPolyDataFilter")
+
+<div>
+
+[Calculator2]: ../docs/gallery/Calculator2.jpg
+[ClipClosedSurface]: ../docs/gallery/ClipClosedSurface.jpg
+[ContourLoopExtraction]: ../docs/gallery/ContourLoopExtraction.jpg
+[ContourTriangulator]: ../docs/gallery/ContourTriangulator.jpg
+[ImageCropFilter]: ../docs/gallery/ImageCropFilter.jpg
+[ImageMarchingCubes]: ../docs/gallery/ImageMarchingCubes.jpg
+[ImageMarchingSquares]: ../docs/gallery/ImageMarchingSquares.jpg
+[ImageStreamline]: ../docs/gallery/ImageStreamline.jpg
+[OBBTree]: ../docs/gallery/OBBTree.jpg
+[OutlineFilter]: ../docs/gallery/OutlineFilter.jpg
+[ScalarToRGBA]: ../docs/gallery/ScalarToRGBA.jpg
+[TriangleFilter]: ../docs/gallery/TriangleFilter.jpg
+[TubeFilter]: ../docs/gallery/TubeFilter.jpg
+[WarpScalargif]: ../docs/gallery/WarpScalar.gif
+[WindowedSincPolyDataFilter]: ../docs/gallery/WindowedSincPolyDataFilter.gif
+
+## Filters/Sources
+
+<div class="gallery">
+
+[![ArrowSource Example][ArrowSource]](./ArrowSource.html "ArrowSource")
+[![CircleSource Example][CircleSource]](./CircleSource.html "CircleSource")
+[![ConcentricCylinderSource Example][ConcentricCylinderSource]](./ConcentricCylinderSource.html "ConcentricCylinderSource")
+[![ConeSource Example][ConeSource]](./ConeSource.html "ConeSource")
+[![Cone Example][Cone]](./Cone.html "Cone source")
+[![CubeSource Example][CubeSource]](./CubeSource.html "CubeSource")
+[![Cursor3D Example][Cursor3D]](./Cursor3D.html "Cursor3D")
+[![CylinderSource Example][CylinderSource]](./CylinderSource.html "CylinderSource")
+[![LineSource Example][LineSource]](./LineSource.html "LineSource")
+[![PlaneSource Example][PlaneSource]](./PlaneSource.html "PlaneSource")
+[![PointSource Example][PointSource]](./PointSource.html "PointSource")
+[![SLICSource Example][SLICSource]](./SLICSource.html "SLICSource")
+[![SphereSource Example][SphereSource]](./SphereSource.html "SphereSource")
+
+<div>
+
+[ArrowSource]: ../docs/gallery/ArrowSource.jpg
+[CircleSource]: ../docs/gallery/CircleSource.jpg
+[ConcentricCylinderSource]: ../docs/gallery/ConcentricCylinderSource.jpg
+[ConeSource]: ../docs/gallery/ConeSource.jpg
 [Cone]: ../docs/gallery/Cone.jpg
-[CubeAxes]: ../docs/gallery/CubeAxes.jpg
-[DepthTest]: ../docs/gallery/DepthTest.jpg
-[GlyphRotation]: ../docs/gallery/GlyphRotation.jpg
-[LookingGlass]: ../docs/gallery/LookingGlassCone.png
-[Texture]: ../docs/gallery/Texture.jpg
-[TimeSeries]: ../docs/gallery/TimeSeries.gif
-[VR]: ../docs/gallery/VrCone.jpg
+[CubeSource]: ../docs/gallery/CubeSource.jpg
+[Cursor3D]: ../docs/gallery/Cursor3D.gif
+[CylinderSource]: ../docs/gallery/CylinderSource.jpg
+[LineSource]: ../docs/gallery/LineSource.jpg
+[PlaneSource]: ../docs/gallery/PlaneSource.jpg
+[PointSource]: ../docs/gallery/PointSource.jpg
+[SLICSource]: ../docs/gallery/SLICSource.jpg
+[SphereSource]: ../docs/gallery/SphereSource.gif
+
+## Interaction

```

Comment:
Merge `Interaction` with `Picking/Selecting` and rename into a unique `Interaction/Picking/Selecting`